shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -93,7 +95,7 @@ @interface FlutterViewController () <FlutterBinaryMessenger, UIScrollViewDelegat
 // Trackpad rotating
 @property(nonatomic, retain)
     UIRotationGestureRecognizer* rotationGestureRecognizer API_AVAILABLE(ios(13.4));
-
+@property(nonatomic, retain) UIPencilInteraction* pencilInteraction API_AVAILABLE(ios(13.4));
```

Comment:
This is manually reference counted code, so you'll need to manually release `_pencilInteraction` (the backing instance variable that gets synthesized from a property declaration).
```
/Volumes/Work/s/w/ir/cache/builder/src/flutter/shell/platform/darwin/ios/framework/Source/FlutterViewController.mm:891:3: error: The '_pencilInteraction' ivar in 'FlutterViewController' was retained by a synthesized property but not released before '[super dealloc]' [clang-analyzer-osx.cocoa.Dealloc,-warnings-as-errors]
  [super dealloc];
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8790504096978028977/+/u/test:_lint_ios_debug_sim/stdout

So in the `dealloc` method below, add:
```objc
  [_pencilInteraction release];
```